### PR TITLE
Finished

### PR DESCRIPTION
**Pull request by John42506176Linux**
Screenshot 2020-06-03 at 10 15 50 AM
Screenshot 2020-06-03 at 10 15 46 AM
Screenshot 2020-06-03 at 10 15 41 AM

### CODE REVIEW

File: portfolio/src/main/webapp/styles/style.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+
+.highlight{
+    width:30%;
+    height:40%;
+    color: black;
+    font-weight: bold;
+    display: block;
+    text-align: center;
+    margin: 0 auto;
+  position: relative;
```
Comment (yukyhaha) [inline portfolio/src/main/webapp/styles/style.css]: nit: indent, also please order alphabetically by field name

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,106 @@
...
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+</head>
+<body>
+  <nav class="navbar navbar-expand-lg navbar-dark bg-primary">
+    <a class="navbar-brand" href="#"> John's Portfolio </a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
```
Comment (yukyhaha) [inline portfolio/src/main/webapp/index.html:14]: nit: for those component that too long and overflow the page, maybe wrap it.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,18 +1,106 @@
...
+ 
+<html  lang = "en">
+<head>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel ="stylesheet" href = "styles/style.css">
+  <title> John's cool</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
```
Comment (yukyhaha) [inline portfolio/src/main/webapp/index.html]: nit: indent here and below

Comment (yukyhaha) [inline portfolio/src/main/webapp/index.html]: curious what does those libs do

**Review by yukyhaha — COMMENTED** [review]
just some minor nit, otherwise it looks good overall

**Comment by John42506176Linux** [comment]
I just added a new commits to fix those issues(though not sure how to wrap the longer lines).

**Comment by John42506176Linux** [comment]
The next thing added was for the week 3 portion.

**Review by yukyhaha — APPROVED** [review]

**Pull request merged**